neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -111,6 +111,11 @@ type VirtualMachineSpec struct {
 	// Override for normal neonvm-runner image
 	// +optional
 	RunnerImage *string `json:"runnerImage,omitempty"`
+
+	// Enable SSH on the VM. It works only if the VM image is built using VM Builder that
+	// has SSH support (TODO: mention VM Builder version).
+	// +optional
```

Comment:
let's also make this default to `false`?

Then, when we go to release this, there can be a one-line change to make it default to `true`

Reply:
Makes sense. I'll make the change.

Reply:
added the default value: b8de84026b60a658e05673f703946682e5580c57